modules/failure-handler/src/main/java/org/apache/ignite/internal/failure/FailureManager.java

```diff
@@ -299,4 +302,10 @@ private boolean throttleThreadDump(FailureType type) {
 
         return throttle;
     }
+
+    private static class StackTrackCapturingException extends IgniteInternalException {
```

Comment:
Did you mean `StackTraceCapturingException`?

Reply:
Exactly, thanks. Fixed